src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -393,9 +393,9 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_arcsin)
 
 // arccos
 MXNET_OPERATOR_REGISTER_UNARY(arccos)
-.describe(R"code(Inverse cosine, element-wise.
+.describe(R"code(Returns element-wise Inverse trigonometric cosine value of the input.
```

Comment:
Same issues as above. Please fix accordingly:

"Returns the inverse cosine of the input array, computed element-wise."